.idea/libraries/Dart_SDK.xml

```diff
@@ -0,0 +1,19 @@
+<component name="libraryTable">
```

Comment:
this file should not be versioned

Reply:
i don't see a version property in the file. What i should remove?

Reply:
Changed gitignore file